--- PgSQL.py.original	2005-06-20 18:02:39.000000000 +0200
+++ PgSQL.py	2006-01-17 18:37:25.000000000 +0100
@@ -2254,7 +2254,12 @@
     elif hasattr(value, '_quote'):
 	return value._quote()
     elif type(value) is DateTimeType:
-	return "'%s'" % value
+	val = "'%s'" % value
+	# the real fix would be to use value.Format() but getting
+	# the proper timezone information may be tricky,
+	# so, this is a hack
+	val = val.replace(',', '.')
+	return val
     elif type(value) is DateTimeDeltaType:
 	return "'%s'" % dateTimeDelta2Interval(value)
     elif isinstance(value, StringType):
