--- GNUmed-v8/server/bootstrap/bootstrap-latest.sh.orig	2008-01-25 09:06:24.000000000 -0500
+++ GNUmed-v8/server/bootstrap/bootstrap-latest.sh	2008-01-25 09:19:49.000000000 -0500
@@ -6,1 +6,1 @@
-ln -vsn client Gnumed
+ln -s server Gnumed
@@ -25,22 +25,22 @@
 fi ;
 
 
-echo "==========================================================="
-echo "Bootstrapping latest GNUmed database."
-echo ""
-echo "This will set up a GNUmed database of version v${VER}"
-echo "with the name \"gnumed_v${VER}\"."
-echo "It contains all the currently working parts including"
-echo "localizations for countries you don't live in. This does"
-echo "not disturb the operation of the GNUmed client in your"
-echo "country in any way."
-echo "==========================================================="
-echo "1) Dropping old baseline gnumed_v2 database if there is any."
-sudo -u postgres dropdb -i ${PORT_DEF} gnumed_v2
+#echo "==========================================================="
+#echo "Bootstrapping latest GNUmed database."
+#echo ""
+#echo "This will set up a GNUmed database of version v${VER}"
+#echo "with the name \"gnumed_v${VER}\"."
+#echo "It contains all the currently working parts including"
+#echo "localizations for countries you don't live in. This does"
+#echo "not disturb the operation of the GNUmed client in your"
+#echo "country in any way."
+#echo "==========================================================="
+#echo "1) Dropping old baseline gnumed_v2 database if there is any."
+sudo -u postgres dropdb ${PORT_DEF} gnumed_v2
 
 
-echo "=========================="
-echo "2) bootstrapping databases"
+#echo "=========================="
+#echo "2) bootstrapping databases"
 
 # baseline v2
 LOG="${LOG_BASE}/bootstrap-latest-v2.log"
@@ -63,7 +63,7 @@
 	echo "Bootstrapping \"gnumed_v3\" did not finish successfully. Aborting."
 	exit 1
 fi
-echo "Dropping obsoleted staging database gnumed_v2 ..."
+#echo "Dropping obsoleted staging database gnumed_v2 ..."
 sudo -u postgres dropdb ${PORT_DEF} gnumed_v2
 
 # v3 -> v4
@@ -75,7 +75,7 @@
 	echo "Bootstrapping \"gnumed_v4\" did not finish successfully. Aborting."
 	exit 1
 fi
-echo "Dropping obsoleted staging database gnumed_v3 ..."
+#echo "Dropping obsoleted staging database gnumed_v3 ..."
 sudo -u postgres dropdb ${PORT_DEF} gnumed_v3
 
 # v4 -> v5
@@ -87,7 +87,7 @@
 	echo "Bootstrapping \"gnumed_v5\" did not finish successfully. Aborting."
 	exit 1
 fi
-echo "Dropping obsoleted staging database gnumed_v4 ..."
+#echo "Dropping obsoleted staging database gnumed_v4 ..."
 sudo -u postgres dropdb ${PORT_DEF} gnumed_v4
 
 # v5 -> v6
@@ -99,7 +99,7 @@
 	echo "Bootstrapping \"gnumed_v6\" did not finish successfully. Aborting."
 	exit 1
 fi
-echo "Dropping obsoleted staging database gnumed_v5 ..."
+#echo "Dropping obsoleted staging database gnumed_v5 ..."
 sudo -u postgres dropdb ${PORT_DEF} gnumed_v5
 
 # v6 -> v7
@@ -111,7 +111,7 @@
 	echo "Bootstrapping \"gnumed_v7\" did not finish successfully. Aborting."
 	exit 1
 fi
-echo "Dropping obsoleted staging database gnumed_v6 ..."
+#echo "Dropping obsoleted staging database gnumed_v6 ..."
 sudo -u postgres dropdb ${PORT_DEF} gnumed_v6
 
 # v7 -> v8
